homework/week2/index.js

```diff
@@ -0,0 +1,94 @@
+const prompts = require('prompts');
+const mysql = require('mysql');
+const config = require('./connection.js');
+
+const connection = mysql.createConnection(config);
+connection.connect();
+
+async function question() {
+  const questions = [
+    {
+      type: 'select',
+      name: 'question',
+      message: 'Pick a question',
+      choices: [
+        { title: 'What is the capital of country X', value: 'country' },
+        { title: 'List all the languages spoken in the region Y', value: 'region' },
+        { title: 'Find the number of cities in which language Z is spoken', value: 'language' },
+        {
+          title: 'Are there any countries that have A) Same official language B) Same region',
+          value: 4,
+        },
+        {
+          title: 'List all the continents with the number of languages spoken in each continent',
+          value: 5,
+        },
+      ],
+    },
+  ];
+
+  const response = await prompts(questions);
+
+  const question = response.question;
+  let input;
+
+  if (['country', 'region', 'language'].includes(question)) {
+    input = await prompts({
+      type: 'text',
+      name: 'text',
+      message: `Enter the ${question}:`,
+    });
+  }
+  let sql;
+  switch (question) {
+    case 'country':
+      sql = `SELECT city.name
+      FROM city, country
+      WHERE city.id=country.capital 
+      AND country.name = "${input.text}";
```

Comment:
This will work, but it's not very secure. 
What about using Prepared statements (as mentioned in the assignment)? Please check the npm package's documentation for how to achieve this.

Reply:
thank you so much for your review. I've missed out "prepared statement " requirement in the homework. I've also forgotten to add instructions for the second and the third part of the homework, to cap it all i forgot to add/commit ER diagram file. I've fixed all those issues.
part 2: part2-trigger.sql
part 3: odo-erd.png